bin/biomaj-cli.py

```diff
@@ -480,8 +480,7 @@ def main():
                 print("Bank name is missing")
                 sys.exit(1)
             bmaj = Bank(options.bank, options, no_log=True)
-            print('Log file: '+bmaj.config.log_file)
-            res = bmaj.remove_pending(options.release)
+            res = bmaj.remove_pending()
```

Comment:
By the way, old method removed all pending sessions at once:
`self.banks.update({'name': self.name}, {'$set': {'pending': []}})`
My point, if `release` passed as argument, load session for this release from its id and remove it from db and from disk. Otherwise, remove all pending sessions.
What do you think?